sylvia/src/types.rs

```diff
@@ -160,6 +160,135 @@ impl<'a, C: cosmwasm_std::CustomQuery, Contract> From<&'a BoundQuerier<'a, C, Co
     }
 }
 
+pub struct EmptyExecutorBuilderState;
+pub struct ReadyExecutorBuilderState;
+
+/// This structure represents a collection of execution methods for
```

Comment:
One more thing. These doc comments lack the `interface` example which was previously present in the `EmptyExecutorBuilder`. Please add it here.

You can also consider hiding parts of the code so that example on `docs.rs` will be more concise.